### PR TITLE
Separate CVE tiles and tabs for VM update

### PR DESCRIPTION
**Pull request by vjwilson**
## Description

Addresses these items from the bug list:

- Cluster list -> click on a cluster -> bar on right does not have platform/node/image vulns separated
- Cluster list -> click on a cluster -> bar on right does not have platform/node/image components separated
- Cluster list -> click on a cluster -> top right pop out to cluster summary page -> top bar has cluster and CVE, not separated out values

for all the different entities

## Checklist
- [ ] Investigated and inspected CI test results
- [s] Unit test and regression tests added

## Testing Performed

Clusters
clusters

Namespaces
namespaces

Deployments
deployments

Images
images

Image Components
image_components

Node Components
node_components

Image CVEs
image_cves

Node CVEs
node_cves

Platform CVEs
platform_cves

(as an example of tab version)
Clusters - tab version
cluster_tabs_version

### CODE REVIEW

**Review by md2119 — COMMENTED** [review]
I will defer the code review to someone from FE team. As far as the UX is concerned, it would really good to order the entity counts in right-hand panel.
Are the UI e2e test failures related?

File: ui/apps/platform/src/Components/workflow/EntityTabs/EntityTabs.js [hunk 1]
```diff
@@ -38,7 +39,11 @@ const EntityTabs = ({ entityType, activeTab }) => {
             relationshipTypes.CONTAINS,
             workflowState.useCase
         ),
-    ];
+    ]
+        // @TODO: Remove the following filter step once ROX_FRONTEND_VM_UPDATES is ON
+        .filter((match) => {
```
Comment (sachaudh) [inline ui/apps/platform/src/Components/workflow/EntityTabs/EntityTabs.js:44]: nit: Wasn't sure if the word `match` made sense here. I think `containedEntity` makes more sense

File: ui/apps/platform/src/Containers/VulnMgmt/Entity/RelatedEntitiesSideList.js [hunk 2]
```diff
@@ -25,99 +26,67 @@ const RelatedEntitiesSideList = ({ entityType, data, altCountKeyMap, entityConte
     const countKeyMap = { ...defaultCountKeyMap, ...altCountKeyMap };
 
     const matches = getEntityTypesByRelationship(entityType, relationshipTypes.MATCHES, useCase)
+        // @TODO: Remove the following filter step once ROX_FRONTEND_VM_UPDATES is ON
+        .filter((match) => {
```
Comment (sachaudh) [inline ui/apps/platform/src/Containers/VulnMgmt/Entity/RelatedEntitiesSideList.js:30]: nit: This is fine how it is, but to be consistent with the suggestion above, maybe `matchedEntity` makes more sense

**Review by sachaudh — APPROVED** [review]
LGTM. Just a naming suggestion for you

**Pull request merged**